javascript/javascript2/week1/shortestword.js

```diff
@@ -0,0 +1,16 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWord(){
```

Comment:
think of adding a parameter for this function and
```suggestion
function shortestWord(words){
```
and called it with `shortestWord(danishWords )`